client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -609,18 +609,22 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	renderSiteLaunchedTask = ( TaskComponent, baseProps, task ) => {
-		const { translate } = this.props;
+		const { taskStatuses, translate } = this.props;
+		const isEmailVerified = get( taskStatuses, 'email_verified.completed' );
+		const buttonText = isEmailVerified
+			? translate( 'Launch site' )
+			: translate( 'Launch site (once you have verified your email address)' );
```

Comment:
I wonder if we could handle this situation in a more general way where we make it clear that one task is fully dependent on another task being completed prior. That could save us having to sort of explain ourselves in words.